app/components/ClientUtilityFunctions.tsx

```diff
@@ -6,7 +6,7 @@ import { useRouter } from 'next/navigation';
  * 
  * @param isUser - true if current session user matches page, false otherwise
  */
-export function EditProfileAndFollowButton({ user, isUser }: { user: string, isUser: boolean }) {
+export function EditProfileAndFollowButton({ user, isUser }: { user: String, isUser: boolean }) {
```

Comment:
Maybe `isCurrentUser` or something similar? The assumption is that if you are viewing a page you are a user so the variable name doesn't describe much.

Reply:
Agreed agreed.